lit/setup-and-operations/install.lit

```diff
@@ -458,6 +458,21 @@ page.
   }
 }
 
+\section{
+	\title{Configuring Container Placement}{configuring-placement-strategies}
+
+	By default, the container placement strategy used is \code{volume-locality}. To set a different container placement strategy, pass the selected option to the \code{--container-placement-strategy} flag.
```

Comment:
Phrasing is a bit awkward. I'd suggest: Concourse uses the \code{volume-locality} container placement strategy by default. Concourse can be configured to use alternate container placement strategies using the \code{--container-placement-strategy} flag.

Reply:
changes made.